snapshot/lib/snapshot/simulator_launchers/launcher_configuration.rb

```diff
@@ -0,0 +1,48 @@
+
+module Snapshot
+  class SimulatorLauncherConfiguration
+    # both
+    attr_accessor :languages
+    attr_accessor :devices
+    attr_accessor :add_photos
+    attr_accessor :add_videos
+    attr_accessor :clean
+    attr_accessor :erase_simulator
+    attr_accessor :localize_simulator
+    attr_accessor :reinstall_app
+    attr_accessor :app_identifier
+
+    # xcode 8
+    attr_accessor :number_of_retries
+    attr_accessor :stop_after_first_error
+    attr_accessor :output_simulator_logs
+
+    # runner
+    attr_accessor :launch_args_set
+    attr_accessor :output_directory
+
+    def initialize(snapshot_config: nil)
+      @languages = snapshot_config[:languages]
```

Comment:
I was thinking about that... that's not too magical? 🤣 

Java Beans are the best [ducks behind table]